Gemfile.lock

```diff
@@ -616,65 +618,63 @@ GEM
     sawyer (0.9.2)
       addressable (>= 2.3.5)
       faraday (>= 0.17.3, < 3)
-    sentry-rails (5.4.2)
+    sentry-rails (5.16.1)
       railties (>= 5.0)
-      sentry-ruby (~> 5.4.2)
-    sentry-ruby (5.4.2)
+      sentry-ruby (~> 5.16.1)
+    sentry-ruby (5.16.1)
       concurrent-ruby (~> 1.0, >= 1.0.2)
     sentry-user_informer (0.2.0)
       rack
       railties (>= 6.1.0, < 7.1.0)
       sentry-rails (~> 5.4)
-    sexp_processor (4.12.1)
-    single_cov (1.3.2)
-    slop (3.6.0)
+    sexp_processor (4.17.1)
+    single_cov (1.11.0)
     snaky_hash (2.0.1)
       hashie
       version_gem (~> 1.1, >= 1.1.1)
     socksify (1.7.1)
-    soft_deletion (1.6.0)
-      activerecord (>= 4.2.0, < 6.2.0)
+    soft_deletion (1.9.0)
+      activerecord (>= 5.0.0, < 7.2)
     sprockets (3.7.2)
       concurrent-ruby (~> 1.0)
       rack (> 1, < 3)
     sprockets-rails (3.4.2)
       actionpack (>= 5.2)
       activesupport (>= 5.2)
       sprockets (>= 3.0.0)
-    sqlite3 (1.4.2)
-    stackprof (0.2.12)
+    sqlite3 (1.7.2)
+      mini_portile2 (~> 2.8.0)
+    stackprof (0.2.26)
     terminal-table (1.8.0)
       unicode-display_width (~> 1.1, >= 1.1.1)
     thor (1.3.0)
-    tilt (2.0.10)
+    tilt (2.3.0)
     tzinfo (2.0.6)
       concurrent-ruby (~> 1.0)
     uglifier (3.2.0)
       execjs (>= 0.3.0, < 3)
-    unf (0.1.4)
-      unf_ext
-    unf_ext (0.0.8.2)
     unicode-display_width (1.8.0)
     validates_lengths_from_database (0.8.0)
       activerecord (>= 4)
     version_gem (1.1.3)
-    warden (1.2.7)
-      rack (>= 1.0)
-    webmock (3.0.1)
-      addressable (>= 2.3.6)
+    warden (1.2.9)
```

Comment:
### [warden](https://github.com/hassox/warden)

**Patch** version upgrade :chart_with_upwards_trend::small_blue_diamond: 1.2.7 → 1.2.9

[_~~change-log~~, ~~source-code~~, [gem-diff](https://my.diffend.io/gems/warden/1.2.7/1.2.9)_]





<details>
<summary>Commits</summary>

A change of **90** commits. See the full changes on [the compare page](https://github.com/wardencommunity/warden/compare/v1.2.7...v1.2.9).

These are the first 10 commits:
- (0d52540) [Update .travis.yml](https://github.com/wardencommunity/warden/commit/0d525408feea7a2a4e118a06a73cc3d9ebea8c6d)
- (4d016e1) [Update .travis.yml](https://github.com/wardencommunity/warden/commit/4d016e1c3cb6dcecc3200576631bf14864f5995a)
- (cc8d65d) [Update .travis.yml](https://github.com/wardencommunity/warden/commit/cc8d65d198c6682bd04211e553ce7f1c55e0aa84)
- (5773ff6) [Set winning strategy after _run!, in case _run! also sets wi…](https://github.com/wardencommunity/warden/commit/5773ff631b679ed435910070326834a885826205)
- (58bcc6f) [Fix magic comment](https://github.com/wardencommunity/warden/commit/58bcc6f0beb58764b2f07bebd46e919feb16a291)
- (ba59bff) [Update .travis.yml](https://github.com/wardencommunity/warden/commit/ba59bff0462b2d592a9e2540e0ee546e5931935f)
- (79442be) [Update README.textile](https://github.com/wardencommunity/warden/commit/79442beed4c0a978a0583690df7125d7bf7d727b)
- (fa24dcb) [Merge pull request #145 from znz/fix-magic-comment](https://github.com/wardencommunity/warden/commit/fa24dcbf34022d85dce8db51dd11bbbe5a6fddcc)
- (617eaa1) [Update .travis.yml](https://github.com/wardencommunity/warden/commit/617eaa10187f1f48e39427d9598ac30e6e02fb63)
- (5a091ac) [Merge pull request #149 from hassox/fix-bundle-on-travis](https://github.com/wardencommunity/warden/commit/5a091ac6c00eb0842f7356a806c557f9b50852e3)


</details>